README.md

```diff
@@ -11,179 +11,80 @@ The workspace in this repository creates the `libbdkffi` multi-language library
 [bdk] library from the [Bitcoin Dev Kit] project. The `bdk-ffi-bindgen` package builds a tool for 
 generating the actual language binding code used to access the `libbdkffi` library.
 
-Each supported language has its own repository that includes this project as a [git submodule]. 
-The rust code in this project is a wrapper around the [bdk] library to expose it's APIs in a 
-uniform way using the [mozilla/uniffi-rs] bindings generator for each supported target language.
+Each supported language has its own directory. The rust code in this project is a wrapper around the [bdk] library to expose its APIs in a uniform way using the [mozilla/uniffi-rs] bindings generator for each supported target language.
 
 ## Supported target languages and platforms
-
 The below repositories include instructions for using, building, and publishing the native 
 language binding for [bdk] supported by this project.
 
-| Language | Platform     | Repository   |
-| -------- | ------------ | ------------ |
-| Kotlin   | jvm          | [bdk-kotlin] |
-| Kotlin   | android      | [bdk-kotlin] |
-| Swift    | iOS, macOS   | [bdk-swift]  |
-| Python   | linux, macOS | [bdk-python] |
+| Language | Platform     | Directory/Repository |
+| -------- |--------------|----------------------|
+| Kotlin   | JVM          | [bdk-jvm]            |
```

Comment:
Remove the brackets for bdk-jvm, bdk-android, etc. since we don't want to link to the old repos, and the links are removed below.